javascript/javascript2/week1/spiritAnimal.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+    <body>
+        <input type="text" id="inputVal">
+        <button class="btn1">Generate Spiral Animal Name</button>
```

Comment:
Not a very descriptive class name:
`generate-spirit-name-button` is long, but more descriptive.